doc/configuration.rst

```diff
@@ -728,6 +728,31 @@ created with ID 2". We can view the details of this new investigator by entering
 	created  2015-09-09 09:53:28.989481 -0400 EDT
 	modified 2015-09-09 09:53:28.989481 -0400 EDT
 
+Recent versions of MIG have introduced the concept of investigators that can be
+considered MIG administrators. Administrator investigators have the ability to 
+manage manifests and manipulate mig-loader related functionality. If you are going
+to use ``mig-loader`` to keep agents up to date automatically, you should make
+your investigator an administrator. If not, it is not required.
+
+For information on mig-loader see `MIG LOADER`_ documentation.
+
+.. _`MIG LOADER`: loader.rst
+
+To make a user an administrator, set ``isadmin`` to true for your investigator
+in the database. The following is an example for the previously created investigator
+with an ID of 2.
+
+::
+
+    $ sudo -u postgres psql mig
+    psql (9.4.7)
+    Type "help" for help.
+    
+    mig=# UPDATE investigators SET isadmin = true WHERE id=2;
+    UPDATE 1
+    mig=# \q
+    $
```

Comment:
This phase of the configuration happens before the API authentication is enabled. Maybe we could allows any investigator to set or unset the administrator bit when the api auth is disabled, which would solved the chicken and egg issue.
Then we could just manage administrators in the API/Console.
Thoughts?